genestack_client/genestack_metainfo.py

```diff
@@ -31,6 +31,7 @@ class Metainfo(dict):
     CONTACT_PERSON = 'genestack:contactPerson'
     EXTERNAL_LINK = 'genestack:links'
     CREATION_DATE = 'genestack:dateCreated'
+    PARENT_DICTIONARY = 'genestack.dictionary:parent'
```

Comment:
I try to move it and decide to leave it here.

We allow user to specify parent dictionary manually by passing metainfo with it. 

IMHO better place for this key in metainfo